File: Topic Based Problems/01 Fundamentals/Beginner/9 Celsius to Fahrenheit/C++/CelsiustoFahrenheit.cpp
/**
 * @file CelsiustoFahrenheit.cpp
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Celsius to Fahrenheit in C++
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double celsius;

    cout << "Enter temperature in Celsius: ";
    cin >> celsius;

    cout << "Temperature in Fahrenheit is: " << fixed << setprecision(2) << (celsius * 9 / 5) + 32 << endl;

    return 0;
}

File: Topic Based Problems/01 Fundamentals/Beginner/9 Celsius to Fahrenheit/Java/CelsiustoFahrenheit.java
/**
 * @file CelsiustoFahrenheit.java
 * @author SharifdotG
 * @brief Celsius to Fahrenheit in Java
 * @version 1.0
 * @date 2023-09-24
 */

import java.util.Scanner;

public class CelsiustoFahrenheit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        double celsius;

        System.out.print("Enter temperature in Celsius: ");
        celsius = scanner.nextDouble();

        System.out.printf("Temperature in Fahrenheit is: %.2f\n", (celsius * 9 / 5) + 32);

        scanner.close();
    }
}
